src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+/*
+Language: Linden Scripting Language
+Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+Author: Builder's Brewery <buildersbrewery@gmail.com>
+Category: scripting
+*/
+
+function(hljs) {
+
+    var LSL_STRING_ESCAPE_CHARS = {
+        className: 'subst',
+        begin: '\\[\\\\nt"\\]'
+    };
+
+    var LSL_STRINGS = {
+        className: 'string',
+        begin: '"',
+        end: '"',
+        contains: [
+            { begin: '/\\*', end: '\\*/', skip: true },
```

Comment:
I'm pretty sure you don't need to use `skip` here, as it's a hack for some special case of sub-languages. But I don't get what this mode was supposed to do inside strings. Do `/* .. */` comments have a special meaning *inside* strings?